lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -599,6 +599,7 @@ private void informAboutConatinerFailureDuringRestart(final String containerId)
   }
 
   private synchronized void writeToEvaluatorLog(final String entry) throws IOException {
+    /*
```

Comment:
Can you please uncomment this? It should be resolved as part of REEF-85 and not here.